src/errors/parser.ts

```diff
@@ -8,35 +8,35 @@ import { PathParameterError } from "./PathParameterError.js";
 import { RequestParameterError } from "./RequestParameterError.js";
 import { RequestError, RequestResponseBaseError, ResponseError } from "./RequestResponseError.js";
 
-export const getProcessingErrorCodeByStatus = (status: number): string => {
+export const getProcessingErrorCodeByStatus = (status: number, isThreeDsError?: boolean, message?: string): string => {
```

Comment:
Very nitpicky on my part but would it be possible to have `message` before `isThreeDsError`?

Also, feel free to ignore this but adding an `options?: { isThreeDsError?: boolean }` feels more future-proof and would also document the code better than just a boolean.